public/js/login.js

```diff
@@ -0,0 +1,59 @@
+
+let loginBtn;
+
+loginBtn = document.getElementById("loginBtn");
+loginBtn.addEventListener('click', loginHandler);
+
+function loginHandler (event ) {
+    event.preventDefault();
+
+    const email = document.querySelector('#email-login').value.trim();
+    const password = document.querySelector('#password-login').value.trim();
+
+    if (email && password) {
+        fetch('/api/users/login', {
+          method: 'POST',
+          body: JSON.stringify({ email, password }),
+          headers: { 'Content-Type': 'application/json' },
+        });
+
+        document.location.replace('/');
+    
+        // if (response.ok) {
```

Comment:
Why we do not check the response status? what if the user email not exist in database?